README.md

```diff
@@ -1,3 +1,19 @@
 # sellotape
-## Team Members
-Guy Itzhaki, Tarel Madar, Gal Lapid, Alon weissfeld, Omri Rosner
+
+Sellotape is a modern audio podcast platform.
+It introduces an accessible way to listen to podcast and audio streams.
+
+As a podcaster/streamer, you can create your private radio. By accessing your user endpoint, anyone who wants to listen can join your podcast, live djset, or whatever it is you want it to be heard.
+
+As a listener, you can follow your favourite streamers and keep up to date with their streams.
```

Comment:
s/favourite/favorite/

Reply:
Didn't get what you mean :)

Reply:
@alonweissfeld  There is a typo mistake, should be favorite instead of favourite (line 8)

Reply:
Isn't that a British vs American english thing? anyway fixed.

Reply:
This is how you substitute a string by a string in regex. Look it up.